pkgs/tools/graphics/waifu2x-ncnn-vulkan/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, vulkan-headers
+, vulkan-loader
+, glslang
+, libgcc
+, libwebp
+, ncnn
+}:
+
+stdenv.mkDerivation rec {
+  pname = "waifu2x-ncnn-vulkan";
+  version = "20220419";
+
+  src = fetchFromGitHub {
+    owner = "nihui";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-cyPqbVEbASZdbUTHnlC8zDmDMiziwbEL9WDcCdN4GFo=";
+  };
+  sourceRoot = "source/src";
+
+  patches = [
+    # Fixes missing target SPIRV-Tools-opt
+    ./cmakelists.patch
+  ];
+
+  cmakeFlags = [
+    "-DUSE_SYSTEM_NCNN=1"
+    "-DUSE_SYSTEM_WEBP=1"
+    "-DGLSLANG_TARGET_DIR=${glslang}/lib/cmake"
+  ];
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ vulkan-headers vulkan-loader glslang libwebp ncnn ]
+    ++ lib.optional (!stdenv.isDarwin) libgcc;
```

Comment:
On a second look, I think this should be `stdenv.isLinux` instead. Same goes for the other derivation.

Reply:
ping?